### ISSUE TITLE
Validation set restore for resuming training

### ISSUE THREAD

**Issue by ruizhaogit** [tflearn/tflearn#199]
Hi TFlearn Team,

I stopped the program during the traning and want to resume training.  Then I found out that the validation accuracy is increased from 70% to 85%. I realized that the traning set and validation set is shuffled again. My code is like this:
model.fit(trainX, trainY, n_epoch=200, shuffle=None, validation_set=0.1)
Is it possible to restore the splits of the train and validation set, when restore the weights from checkpoints?

Thank you!

Best,
Rui

**Comment by aymericdamien** [comment]
Currently it is not possible. If you want to use a same validation set, you have to create your own one.
Otherwise, you can also change tflearn source code: https://github.com/tflearn/tflearn/blob/master/tflearn/helpers/trainer.py#L642 You have to replace that line with your own shuffling method (that will always shuffle the same way)
